### PR TITLE
Allow application to run without a Raspberry Pi

### PR DESCRIPTION
**Pull request by stephen322**
Hopefully will allow easier development for certain sections of code.
Comes with a web-based LED emulator (alpha).

System/Platform and Hotspot functions moved to platform.py.  So when not running on a Pi will hopefully not mess up the OS.

### CODE REVIEW

File: lib/platform.py [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+    def get_current_connections(self):
+        return False, "Platform disabled", ""
+
+class PlatformRasp():
+    def copy_connectall_script(self):
+        # make sure connectall.py file exists and is updated
+        if not os.path.exists('/usr/local/bin/connectall.py') or \
+                filecmp.cmp('/usr/local/bin/connectall.py', 'lib/connectall.py') is not True:
+            print("connectall.py script is outdated, updating...")
+            copyfile('lib/connectall.py', '/usr/local/bin/connectall.py')
```
Comment (onlaj) [inline lib/platform.py:31]: It tries to copy "connectall.py" into directory that might not exist on other systems.

File: lib/rpi_drivers.py
Comment (onlaj) [inline lib/rpi_drivers.py:1]: There are imports missing, is this on purpose?

Comment (stephen322) [reply lib/rpi_drivers.py:1]: Hmm, such as?

Comment (onlaj) [reply lib/rpi_drivers.py:1]: Never mind. Pycharm showed me missing packages, but I was on local interpreter instead of remote, all good.

**Pull request merged**